R/repair-names.R

```diff
@@ -95,14 +209,14 @@ describe_tidying <- function(orig_name, name) {
   }
 }
 
-#' @rdname tidy_names
-#' @description
-#' `repair_names()` is an older version with different renaming heuristics,
-#' kept for backward compatibility.  New code should prefer `tidy_names()`.
+#' @rdname name-repair
+#' @description `repair_names()` is an older function with different renaming
+#'   heuristics, kept for backward compatibility. Use `set_valid_names()` or
+#'   `set_tidy_names()` in new code.
 #'
 #' @param prefix A string, the prefix to use for new column names.
 #' @param sep A string inserted between the column name and de-duplicating
-#'    number.
+#'   number.
 #' @export
 repair_names <- function(x, prefix = "V", sep = "") {
   if (length(x) == 0) {
```

Comment:
This should probably now be formally deprecated

Reply:
I went with soft deprecation (`message()`) because `dplyr::bind_cols()` calls this.